src/core/packages/chrome/browser/src/project_navigation.ts

```diff
@@ -405,8 +405,15 @@ export interface NavigationTreeDefinition<
    * or "group" items.
    * */
   footer?: Array<RootNavigationItemDefinition<LinkId, Id, ChildrenId>>;
+  /**
+   * Special callout section displayed between the body and footer.
+   * Typically used for promotional or informational content.
+   * */
+  callout?: Array<RootNavigationItemDefinition<LinkId, Id, ChildrenId>>;
```

Comment:
I looked for any alternatives but I'm not seeing any other way to achieve the desired look here. 🙂

This is good for now, but will not work in the future designs that take away screen real estate from the main nav.